doc/content/guides/connecting-gateways/ciscoixm/_index.md

```diff
@@ -0,0 +1,348 @@
+---
+title: "Cisco LoRaWAN Gateway"
+description: ""
+weight: 1
+---
+
+The Cisco LoRaWAN Gateway is a LoRaWAN gateway, whose technical specifications can be found in [the official documentation](https://www.cisco.com/c/en/us/products/routers/wireless-gateway-lorawan/). This page guides you to connect it to {{% tts %}}.
+
+## Prerequisites
+
+1. User account on {{% tts %}} with rights to create Gateways.
+2. Cisco LoRaWAN Gateway with latest firmware version `2.0.32`.
+3. A console cable from USB to RJ45.
+
+## Registration
+
+Create a gateway by following the instructions for the [Console]({{< ref "/guides/getting-started/console#create-gateway" >}}) or the [CLI]({{< ref "/guides/getting-started/cli#create-gateway" >}}).  
+The **EUI** is derived from the **MAC_ADDRESS** that can be found on the back panel of the gateway. To get the **EUI** from the **MAC_ADDRESS** insert FFFE after the first 6 characters to make it a 64bit EUI.
+
+If you followed the [Getting Started guide]({{< ref "/guides/getting-started" >}}) the **Gateway Server Address** is the same as what you use instead of `thethings.example.com`.
+
+## Configuration
+  
+Plug the RJ45 end of the cable in the Console port of the gateway, and the USB port to your computer.  
+Then, if you are using MacOS or Linux, connect to the Gateway by opening a terminal and a executing the following commands.  
+Use PuTTy if you are using Windows.
+
+```bash
+$ ls /dev/tty.usb*
+```
+
+> Note: This displays the list of available USB serial devices.
+
+Once you’ve found the once matching the Cisco console, connect using the following command
+
+```bash
+$ screen <device> 115200 # exemple: screen /dev/tty.usbserial-AO001X6M 115200
+```
+You are now in the gateway's shell, called `standalone mode`.
+
+### System setup
+
+First you need to enable the privileged mode.
+
+```
+Gateway> enable
+```
+
+#### Network
+
+To configure your Cisco Gateway to your network, type the following commands:
+
+```
+Gateway# configure terminal 
```

Comment:
So the input switches from `Gateway>` to `Gateway#` ?

Reply:
Yes because in the privileged mode after the command `enable` I think it is easier for them if we show them how it will be displayed on the screen:
```
Gateway>
Gateway>enable
Gateway#configure terminal 
Gateway(config)#
Gateway(config)#
Gateway(config)#
```